README.md

```diff
@@ -0,0 +1,8 @@
+amz-review-scraper
```

Comment:
You always want to have a README.md at the root of the project, since it's used as the homepage for project. Also you never know who is going to be looking over it, and if it has a README, they may stop try it out, and / or contribute!

Also when you create a repo with Github, they even give you a check box to create it with a Repo